test/cql-pytest/test_materialized_view.py

```diff
@@ -1189,3 +1192,63 @@ def test_base_partition_deletion_with_low_timestamp(cql, test_keyspace):
             # Verify we get only the row with the newer timestamp
             assert [2] == [x.c for x in cql.execute(f"SELECT c FROM {table} WHERE p=0")]
             assert [2] == [x.c for x in cql.execute(f"SELECT c FROM {mv} WHERE p=0")]
+
+# Verify that creating a materialized view with an ID that is already used fails.
+def test_creating_mv_with_existing_id(cql, test_keyspace):
+    # Returns the ID of a given CF. `cf_full_name` is of form `ks_name.cf_name`.
+    def get_id_of_cf(cf_full_name: str) -> str:
+        create_stmt = cql.execute(f'DESC TABLE {cf_full_name} WITH INTERNALS').one().create_statement
```

Comment:
Please make sure this test passes on Cassandra too (again, if you don't know how to do it, let me know, but it's explained in test/cql-pytest/README.md). If I remember correctly, in Cassandra DESC TABLE is not allowed on a view, and you must use DESC MATERIALIZED VIEW.